app/services/analytics.rb

```diff
@@ -28,12 +30,14 @@ def uuid
   end
 
   # rubocop:disable Metrics/LineLength
+  ACCOUNT_CREATION_INTRO_VISIT = 'Account creation intro visited'.freeze
   AUTHENTICATION_MAX_2FA_ATTEMPTS = 'Authentication: user reached max 2FA attempts'.freeze
   EMAIL_AND_PASSWORD_AUTH = 'Email and Password Authentication'.freeze
   EMAIL_CHANGE_REQUESTED = 'Email Change: requested'.freeze
   EMAIL_CHANGED_TO_EXISTING = 'Email Change: user attempted to change their email to an existing email'.freeze
   EMAIL_CONFIRMATION = 'Email Confirmation'.freeze
   IDV_FAILED = 'IdV: Failed'.freeze
+  IDV_INTRO_VISIT = 'IdV Intro Visited'.freeze
```

Comment:
Currently, we don't really have any consistency. The original colon format you introduced was because we were generating multiple events for what is really one event. As I've been refactoring, I've been consolidating these events into one, and in the process removing the need for a colon. Having said that, there are some concepts that require multiple events, and IdV is probably one of them.